client/src/components/rooms/RoomsMeetingExperience.tsx

```diff
@@ -45,8 +58,28 @@ export const RoomsMeetingExperience = (props: RoomsMeetingExperienceProps): JSX.
     _createAdapters();
   }, [credential]);
 
+  if (credential === undefined) {
+    return <>Failed to construct credential. Provided token is malformed.</>;
+  }
   if (callAdapter) {
     const formFactorValue = new MobileDetect(window.navigator.userAgent).mobile() ? 'mobile' : 'desktop';
+
+    if (renderPostCall && postCall && userRole === RoomParticipantRole.attendee) {
+      return (
+        <Survey
+          callId={callId}
+          acsUserId={userId}
+          meetingLink={locator.roomId}
+          theme={fluentTheme}
+          data-testid="Survey"
```

Comment:
we are not using this data-testid anywhere, so we can remove this

Reply:
done!